test/end-to-end/pages/screeners/startapply.js

```diff
@@ -1,16 +1,9 @@
 'use strict';
-const content = require('app/resources/en/translation/screeners/startapply');
+const pageUnderTest = require('app/steps/ui/screeners/startapply');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?